src/go/rdctl/cmd/snapshot.go

```diff
@@ -60,6 +61,19 @@ func exitWithJsonOrErrorCondition(e error) error {
 	return errors.Join(snapshotErrors...)
 }
 
+func getSnapshotId(manager snapshot.Manager, desiredName string) (string, error) {
```

Comment:
Why is this not a method on the manager object?

```suggestion
func (manager snapshot.Manager) getSnapshotId(desiredName string) (string, error) {
```

Reply:
Good idea